extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.js

```diff
@@ -0,0 +1,654 @@
+          />
+        </svg>
+      </div>
+    </div>
+  `;
+};
+
+/**
+ * Generates the template for the add device dialog.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+const buildAddDevicePopupTemplate = (element) => {
+  const html = htmlFor(element);
+  return html`
+    <div class="i-amphtml-story-dev-tools-device-popup-bg">
+      <div
+        class="i-amphtml-story-dev-tools-device-popup-container i-amphtml-story-dev-tools-device-popup-add-devices"
+      >
+        <h1>Mobile</h1>
+        <div class="i-amphtml-story-dev-tools-device-popup-mobile"></div>
+        <h1>Tablet</h1>
+        <div class="i-amphtml-story-dev-tools-device-popup-tablet"></div>
+        <h1>Desktop</h1>
+        <div class="i-amphtml-story-dev-tools-device-popup-desktop"></div>
+        <svg
+          title="cross"
+          class="i-amphtml-story-dev-tools-device-popup-close"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 24 24"
+          fill="white"
+          width="18px"
+          height="18px"
+        >
+          <path
+            d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
+          />
+        </svg>
+      </div>
+    </div>
+  `;
+};
+
+const DEFAULT_DEVICES = 'iphone11discover;oneplus5t;pixel2';
+
+const ALL_SCREEN_SIZES = [
+  {
+    'name': 'Pixel 2',
+    'width': 411,
+    'height': 605,
+    'deviceHeight': 731,
+  },
+  {
+    'name': 'Pixel 3',
+    'width': 411,
+    'height': 686,
+    'deviceHeight': 823,
+  },
+  {
+    'name': 'iPhone 8 (Browser)',
+    'width': 375,
+    'height': 554,
+    'deviceHeight': 667,
+  },
+  {
+    'name': 'iPhone 8 (Native)',
+    'width': 375,
+    'height': 632,
+    'deviceHeight': 667,
+  },
+  {
+    'name': 'iPhone 11 (Browser)',
+    'width': 414,
+    'height': 724,
+    'deviceHeight': 896,
+  },
+  {
+    'name': 'iPhone 11 (Native)',
+    'width': 414,
+    'height': 795,
+    'deviceHeight': 896,
+  },
+  {
+    'name': 'iPhone 11 Pro (Browser)',
+    'width': 375,
+    'height': 635,
+    'deviceHeight': 812,
+  },
+  {
+    'name': 'iPhone 11 Pro (Native)',
+    'width': 375,
+    'height': 713,
+    'deviceHeight': 812,
+  },
+  {
+    'name': 'iPad (Browser)',
+    'width': 810,
+    'height': 1010,
+    'deviceHeight': 1080,
+  },
+  {
+    'name': 'OnePlus 5T',
+    'width': 455,
+    'height': 820,
+    'deviceHeight': 910,
+  },
+  {
+    'name': 'OnePlus 7 Pro',
+    'width': 412,
+    'height': 743,
+    'deviceHeight': 892,
+  },
+  {
+    'name': 'Desktop 1080p',
+    'width': 1920,
+    'height': 1080,
+    'deviceHeight': 1080,
+  },
+];
+
+/**
+ * Method that returns lowercase and remove non-alphanumeric, used on matching hashString.
+ * @param {string} name
+ * @return {string}
+ */
+function simplifyDeviceName(name) {
+  return name.toLowerCase().replace(/[^a-z0-9]/gi, '');
+}
+
+/**
+ * Get devices from the queryHash into list of devices
```

Comment:
Looks like you get them from the element attribute that you formatted yourself, plz update the comment/param names